pkg/app/mongodb.go

```diff
@@ -63,9 +63,6 @@ func NewMongoDB(name string) HelmApp {
 			Chart:    "mongodb",
 			Values: map[string]string{
 				"architecture":     "replicaset",
-				"image.registry":   "ghcr.io",
-				"image.repository": "kanisterio/mongodb",
-				"image.tag":        "v9.99.9-dev",
```

Comment:
@shahpratikr should we remove these container images from our goreleaser workflow as well? If we are not consuming them anymore.

Reply:
@viveksinghggits this image is used in blueprints right?
```
./examples/mongodb/blueprint-v2/mongo-blueprint.yaml:        image: ghcr.io/kanisterio/mongodb:0.85.0
./examples/mongodb/blueprint-v2/mongo-blueprint.yaml:        image: ghcr.io/kanisterio/mongodb:0.85.0
./examples/mongodb/blueprint-v2/mongo-blueprint.yaml:        image: ghcr.io/kanisterio/mongodb:0.85.0
```

Reply:
If we don't use them in application, do we need them in any of the blueprints? Are we just using them in the v2 blueprints?

Reply:
We are using this image at multiple places. This was just the snippet from grep command. If you run `grep "kanisterio/mongodb" -rw .` inside kanister repo, you will get all the files where we are using these images

Reply:
Like discussed let's raise another PR to with changes to make sure we remove the images from goreleaser that are not used anymore, and app-tools image can be used instead.